examples/example_05_dots_segmentation.py

```diff
@@ -0,0 +1,179 @@
+"""
+DOTS white matter segmentation
+==============================
+
+This example shows how to perform white matter segmentation using Diffusion
+Oriented Tract Segmentation (DOTS) algorithm [1]_:
+
+1. Downloading DOTS atlas prior informationg using
+    :func:`nighres.data.download_DOTS_atlas`
+2. Downloading an example DTI data set using
+    :func:`nighres.data.download_DTI_2mm`
+3. Segmenting white matter in DTI images into major tracts using 
+    :func:`nighres.brain.dots_segmentation` [1]_
+4. Visualizing the results using matplotlib
+
+"""
+
+############################################################################
+# Import and download
+# -------------------
+# First we import ``nighres`` and ``os`` to set the output directory. Make sure 
+# to run this file in a  directory you have write access to, or change the 
+# ``out_dir`` variable below. We can downloadthe  DOTS atlas priors and an 
+# example DTI dataset using the following command. The registration step of the 
+# DOTS function relies on ``dipy``, so make sure you have installed it 
+# (https://nipy.org/dipy/).
+
+import os
+import nighres
+
+in_dir = os.path.join(os.getcwd(), 'nighres_examples/data_sets')
+out_dir = os.path.join(os.getcwd(), 'nighres_examples/dots_segmentation')
+nighres.data.download_DOTS_atlas()
+DTI_data, DTI_mask = nighres.data.download_DTI_2mm(in_dir)
```

Comment:
The download function returns a dictionary with entries 'dti' and 'mask, not two outputs. Like this, the segmentation below fails.